sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/StringExpressionsSuite.scala

```diff
@@ -559,9 +559,6 @@ class StringExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
     // scalastyle:on
     checkEvaluation(StringTrim(Literal("a"), Literal.create(null, StringType)), null)
     checkEvaluation(StringTrim(Literal.create(null, StringType), Literal("a")), null)
-
-    checkEvaluation(StringTrim(Literal("yxTomxx"), Literal("xyz")), "Tom")
-    checkEvaluation(StringTrim(Literal("xxxbarxxx"), Literal("x")), "bar")
```

Comment:
can we still keep the tests with the actual parameter order?

Reply:
sure.

Reply:
We need to update the test

Reply:
The order changed in #24902 didn't has affect on constructor `StringTrim(srcStr, trimStr)` and so test results don't change before/after #24902. So, I feel it's unnecessary to update parameter order here. And we do update parameter order in sql test(`string-expressions.sql`). WDYT?

Reply:
What about we just revert here cleanly and add a test in a separate PR?

Reply:
sounds good.